### PR TITLE
[ADD] add module product_life_period

### PR DESCRIPTION
**Pull request by laetitia-gangloff**
The goal of this module is to define life period on product and set product in state end at the end of the period.

### CODE REVIEW

File: product_life_period/i18n/fr.po [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+#. module: product_life_period
+#: model:ir.actions.act_window,name:product_life_period.action_open_life_period_products
+#: model:ir.actions.act_window,name:product_life_period.product_life_period_action
+#: model:ir.ui.menu,name:product_life_period.product_life_period_menu
+#: view:product.life.period:product_life_period.product_life_period_view_form
+#: view:product.life.period:product_life_period.product_life_period_view_search
+#: view:product.life.period:product_life_period.product_life_period_view_tree
+msgid "Product Life Period"
+msgstr "Période de vie de l'articles"
```
Comment (Cedric-Pigeon) [inline product_life_period/i18n/fr.po]: Remove "s" on article

File: product_life_period/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
```
Comment (Cedric-Pigeon) [inline product_life_period/__openerp__.py]: Please use simple headers following OCA templates
https://github.com/OCA/maintainer-tools/blob/master/template/module/__openerp__.py#L3

**Comment by Cedric-Pigeon** [comment]
:+1:

File: product_life_period/views/product_life_period_views.xml [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+                                <field name="name"/>
+                            </h1>
+                        </div>
+                        <div class="oe_right oe_button_box">
+                            <button
+                                class="oe_inline oe_stat_button"
+                                type="action"
+                                name="%(action_open_life_period_products)d"
+                                icon="fa-cubes">
+                                <field name="products_count" string="Products" widget="statinfo" />
```
Comment (Cedric-Pigeon) [inline product_life_period/views/product_life_period_views.xml:66]: It should be great to pass the current period in context as default value on product action

**Comment by laetitia-gangloff** [comment]
@Cedric-Pigeon I add a context on action to set default period life

**Pull request closed without merging**